tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_optimizer.py

```diff
@@ -39,7 +40,7 @@
 from tensorflow.python.training import slot_creator
 
 
-def DynamicEmbeddingOptimizer(self, bp_v2=None):
+def DynamicEmbeddingOptimizer(self, bp_v2=None, dist_sync=None):
   """ An optimizer wrapper to make any TensorFlow optimizer capable of training
   Dynamic Embeddding Variables.
 
```

Comment:
Please add comments for the two parameters, I give you the bp_v2's comment:
"""
     Args:
      bp_v2: If True, updating parameters will use *updating* instead of *setting*, which solves
        the race condition problem among workers during back-propagation in large-scale
        distributed asynchronous training.  
        Reference: [Jiang et al., 2020](https://www.usenix.org/conference/osdi20/presentation/jiang)
      dist_sync: bulabula....
"""

Reply:
Please pay attention to the indentation

Reply:
Thanks, done.